wifiphisher/pywifiphisher.py

```diff
@@ -538,7 +539,8 @@ def start(self):
                 if self.network_manager.is_interface_valid(
                         args.internetinterface, "internet"):
                     internet_interface = args.internetinterface
-                    self.network_manager.unblock_interface(internet_interface)
+                    if pyw.iswireless(internet_interface):
+                        self.network_manager.unblock_interface(internet_interface)
```

Comment:
There's no need to check if the interface is wireless at all, right? In any case, you can simply `self.network_manager.unblock_interface(internet_interface)`.